packages/fields-wysiwyg-tinymce/index.js

```diff
@@ -0,0 +1,20 @@
+const { Text } = require('@keystone-alpha/fields');
+const path = require('path');
+
+function bindStaticMiddleware(server) {
+  const tinymce = require.resolve('tinymce');
+  const tinymcePath = tinymce.substr(0, tinymce.lastIndexOf('/'));
+  server.app.use('/tinymce', server.express.static(tinymcePath));
```

Comment:
This doesn't take into account any `adminPath` the user may have set (for example; in the blog project, it's set to `/admin` (the default)).

This is non-blocking. Have added it to the list of follow-up tasks in #936.

Reply:
Should we make it easy for migration, use `static` path from config? (similar to v4)